src/sentry/static/sentry/app/stores/committerStore.tsx

```diff
@@ -0,0 +1,106 @@
+import Reflux from 'reflux';
+
+import CommitterActions from 'app/actions/committerActions';
+import {Committer} from 'app/types';
+
+type CommitterStoreInterface = {
+  state: {
+    // Use `getCommitterStoreKey` to generate key
+    [key: string]: {
+      committers?: Committer[];
+      committersLoading?: boolean;
+      committersError?: Error;
+    };
+  };
+
+  load(orgSlug: string, projectSlug: string, eventId: string): void;
+  loadSuccess(
+    orgSlug: string,
+    projectSlug: string,
+    eventId: string,
+    data: Committer[]
+  ): void;
+  loadError(orgSlug: string, projectSlug: string, eventId: string, error: Error): void;
+
+  get(
+    orgSlug: string,
+    projectSlug: string,
+    eventId: string
+  ): {
+    committers?: Committer[];
```

Comment:
you have the same types in different places. Maybe you could create a reusable type?

Reply:
I've removed the repeated type in `withCommitters.tsx`.

Reply:
Maybe you could create a type like:

```
type CommitersData = {
    committers?: Committer[];
    committersLoading?: boolean;
    committersError?: Error;
}
``` 

and use it here and here https://github.com/getsentry/sentry/blob/057c6a1a13ae83882e89bf2da5dc3e0e933cdefa/src/sentry/static/sentry/app/stores/committerStore.tsx#L10

Reply:
I still would prefer to have a reusable type 🙂